src/charmed_kubeflow_chisme/components/pebble_component.py

```diff
@@ -340,8 +342,8 @@ def get_status(self) -> StatusBase:
         return ActiveStatus()
 
 
-def get_pebble_ready_event_from_charm(charm: CharmBase, container_name: str) -> str:
-    """Returns the pebble-ready event for a given container_name."""
+def get_event_from_charm(charm: CharmBase, container_name: str, event_name: str) -> str:
```

Comment:
suggestion: add a unit test for this function

Reply:
Added on in `tests/units/component/test_pebble_component.py`